lib/models/application.coffee

```diff
@@ -43,6 +46,12 @@ getApplicationModel = (deps, opts) ->
 
 	exports._getId = getId
 
+	normalizeDeviceInfo = (application) ->
```

Comment:
Can we call this `normalizeApplication`? Makes it clearer how we should use it in future, and it'd be a useful place for other similar normalizations, because I'm sure we'll want some more.